tests/pfcwd/test_pfcwd_timer_accuracy.py

```diff
@@ -350,21 +350,18 @@ def retrieve_timestamp(self, pattern):
             syslog_msg = self.dut.shell(cmd)['stdout']
 
             # Regular expressions for the two timestamp formats
-            regex1 = re.compile(r'^[A-Za-z]{3} \d{2} \d{2}:\d{2}:\d{2}\.\d{6}')
-            regex2 = re.compile(r'^\d{4} [A-Za-z]{3}\s{1,2}\d{1,2} \d{2}:\d{2}:\d{2}\.\d{6}')
-
-            if regex1.match(syslog_msg):
-                timestamp = syslog_msg.replace('  ', ' ').split(' ')[2]
-            elif regex2.match(syslog_msg):
-                timestamp = syslog_msg.replace('  ', ' ').split(' ')[3]
+            regex = re.compile(r'\b[A-Za-z]{3}\s{1,2}\d{1,2} \d{2}:\d{2}:\d{2}\.\d{6}\b')
```

Comment:
for month part, No restriction on valid month names:
    The pattern [A-Za-z]{3} allows any three-letter combination (e.g., Xyz), instead of restricting it to valid month names like Jan to Dec.
Suggest to use (?:Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec)

Reply:
Thanks for your suggestion.
Based on the PR https://github.com/sonic-net/sonic-mgmt/pull/11806.
The timestamp format should be FMT_YEAR = "%Y %b %d %H:%M:%S.%f", which used %b, abbreviated mode. The month field uses 3 abbreviated letters.
The timestamp is at the beginning of the logging, so here I think use 3\s should be fine to search the month.